.idea/copyright/zeebe_copyright.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under&#10;one or more contributor license agreements. See the NOTICE file distributed&#10;with this work for additional information regarding copyright ownership.&#10;Licensed under the Zeebe Community License 1.1. You may not use this file&#10;except in compliance with the Zeebe Community License 1.1.&#10;" />
+    <option name="myName" value="zeebe-copyright" />
```

Comment:
:wrench: Maybe rename file and shorthand to _Zeebe Community License_  which seems more precise

Reply:
If by shorthand you mean `myName` then unfortunately there is no option to change it. I can rename it manually, but IntelliJ will override it when any changes are made to the copyright settings.